docs/index.rst

```diff
@@ -12,6 +12,19 @@ Contents:
    api-reference/index
 
 
+Warehouse is a new code base that implements a Python package index. It is being actively developed and the plan is that it will eventually power https://pypi.python.org/pypi (PyPI) and replace an older code base that is currently powering PyPI. You can see Warehouse in production at https://warehouse.python.org
+
+Warehouse is built on top of the Pyramid web application framework and uses PostgreSQL, Elasticsearch, etc.
```

Comment:
I'd probably just delete this sentence. We don't currently use Elasticsearch (though it's likely it will for the search), but more importantly I don't think it's really that pertinent in a high level "What is Warehouse" overview. Describing the stack could potentially go in a "Technology" section somewhere though if folks think it's generally useful.